telegram/ext/filters.py

```diff
@@ -634,7 +634,7 @@ def __init__(self, pattern: Union[str, Pattern]):
             self.name = f'Filters.regex({self.pattern})'
 
         def filter(self, message: Message) -> Optional[Dict[str, List[Match]]]:
-            """"""  # remove method from docs
+            """ """  # remove method from docs
```

Comment:
uhm, what's all this about? :D

Reply:
Ah, it's done by black formatter

Reply:
mh, that's odd. it doesn't do it on my machine, so I'll revert that for the time being, as that's not really related to the pr :D